invenio_app_ils/ui/src/invenio_app_ils/common/api/documentRequests/__tests__/documentRequst.js

```diff
@@ -6,6 +6,6 @@ describe('Document request query builder tests', () => {
       .query()
       .withState('PENDING')
       .qs();
-    expect(query).toEqual('state:"PENDING"');
```

Comment:
there is a typo in the name of this file 👀

Reply:
I think this is fixed now.